client/hosting-overview/components/style.scss

```diff
@@ -0,0 +1,148 @@
+.hosting-overview__top-cards-wrapper {
+	display: flex;
+	gap: 33px;
+	margin-bottom: 38px;
+}
+
+.hosting-overview__card {
+	border-radius: 4px;
+	border: 1px solid var(--studio-gray-5);
```

Comment:
This border conflicts with the box-shadow of the `.card` element and makes it look like the border is bigger than the one proposed in Figma.

We should either remove the box-shadow or the border:

<img width="968" alt="Screenshot 2024-04-12 at 14 06 58" src="https://github.com/Automattic/wp-calypso/assets/1233880/3584ede8-84c1-45bc-88d1-16efcdcaae85">

<img width="965" alt="Screenshot 2024-04-12 at 14 07 22" src="https://github.com/Automattic/wp-calypso/assets/1233880/a711b27f-0afe-4be9-bca8-dcc16840bc44">

Reply:
Good catch! Addressed on https://github.com/Automattic/wp-calypso/pull/89476/commits/f0b90c169a97b4987a2923d77a93825b31c2345f